modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/dto.ftl

```diff
@@ -235,6 +237,10 @@ public class ${schemaName} <#if dtoParentClassName?has_content>extends ${dtoPare
 		<#if propertySchema.jsonMap>
 			@JsonAnySetter
 		</#if>
+		<#if propertySchema.jsonString>
+			@JsonDeserialize(using = JSONStringStdDeserializer.class)
+
```

Comment:
I still ser the extra space!

Reply:
ad804254611c83a72b435da428e9dec46f1e37fb